### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by sebashwa**
Some basic tests for the functionality of selene itself.

### CODE REVIEW

File: test/unit/index.test.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
+var driver = require('selenium-webdriver');
+var selene = require('../..');
+var expect = require('unexpected');
+const se = selene({ browser: 'phantomjs', base: 'file://' });
+const FIXTURE_URL = process.cwd() + '/test/fixtures/test.html';
```
Comment (fgnass) [inline test/unit/index.test.js]: Ah I missed that, sorry. What about `path.resolve(__dirname, '../fixtures/test.html')`?

Comment (sebashwa) [inline test/unit/index.test.js]: Yeah, that is probably a bit safer! :+1: As a side node: if you run the tests via `npm run test-unit` `process.cwd()` seems to resolve fine, even when called from a subdir.. but as I said before, it feels a bit awkward :relieved:

File: package.json [hunk 2]
```diff
@@ -5,6 +5,9 @@
   "repository": "LiquidLabsGmbH/selene",
   "author": "Felix Gnass <f.gnass@liquidlabs.de>",
   "license": "MIT",
+  "scripts": {
+    "test-unit": "./node_modules/.bin/mocha ./test/unit/**/*.test.js --reporter nyan"
```
Comment (fgnass) [inline package.json]: Shouldn't we call the script just `test`? This would allow us to run `npm test` instead of `npm run test`.

Comment (sebashwa) [inline package.json]: Sure! We just separated them, because of the other test file (test/index.js)

File: test/unit/index.test.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+      it('waits for the respective element to appear in the DOM', function() {
+        se.find('#delayed_wrapper').click();
+
+        var waitingPromise = se.wait('#exists_soon', 501, 'description');
+        return expect(waitingPromise, 'when fulfilled', 'to be a', driver.WebElement);
+      });
+    });
+
+
+    describe.skip('when the condition is a function', function() {
```
Comment (sebashwa) [inline test/unit/index.test.js]: Should we keep this?
We could not get it running and then started working something else.

**Pull request merged**